dascore/io/optodas/__init__.py

```
@@ -6,4 +6,4 @@
 More info here: https://web.asn.com/
 """
 from __future__ import annotations
-from .core import OptoDASV8
+from .core import OptoDASV8, OptoDASV10
```

Ruff (0.11.9)

`.core.OptoDASV8` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)